Makefile

```diff
@@ -6,11 +6,12 @@ VERSION ?= $(error VERSION environment variable must be set)
 
 # Run cargo check
 check: deps-build
-	cargo check --workspace --tests --benches --lib --bins --examples
+	cargo clippy --all --all-targets -- -D warnings
```

Comment:
Hey @macro-ss I don't think replacing check with clippy is exactly what we want.  These two do different things right?  I think we should do both.

Reply:
ah - I might misunderstood what this [comment meant](https://github.com/filecoin-project/builtin-actors/issues/66#issuecomment-1212606981), sorry!

Reply:
@ZenGround0 I requested this explicitly https://github.com/filecoin-project/builtin-actors/issues/66#issuecomment-1212606981

Reply:
ah my bad